### ISSUE TITLE
Updating to match latest dask.distributed

### ISSUE THREAD

**Issue by SarvaPulla** [tethysplatform/tethys_dask_scheduler#4]
There are several breaking changes with the latest version of dask.distributed. Here are a few things that have been deprecated.

- from distributed.utils import ignoring (Can be replaced with from contextlib import suppress)
- from distributed.preloading import preload_modules (Not sure what the replacement is)
- from distributed.dashboard import BokehScheduler (Doesn't exist anymore)
 
Not sure the last version that works.

**Comment by swainn** [comment]
We are using the first two here: https://github.com/tethysplatform/tethys_dask_scheduler/blob/96f7abca51035a887f127cbd1cb7d1590e27bb2c/tethys_dask_scheduler/dask_scheduler.py#L16-L19

And the last one here:
https://github.com/tethysplatform/tethys_dask_scheduler/blob/96f7abca51035a887f127cbd1cb7d1590e27bb2c/tethys_dask_scheduler/dask_scheduler.py#L120

**Comment by swainn** [comment]
@SarvaPulla I've finally been able to get around to issuing a fix for this issue. It has been released in version 2.0.0: https://pypi.org/project/tethys-dask-scheduler/

The usage had changed slightly, so pay attention to the instructions in the README. I'm not sure if version 2.0.0 will work with the version of Dask that is installed in Tethys 3.*, but I'm guessing it should. I completed this upgrade in preparation for Tethys 4.

Note: the package won't be available on conda forge until they merge the feedstock PR: https://github.com/conda-forge/staged-recipes/pull/17640
